pkg/controller/deployment.go

```diff
@@ -451,7 +451,7 @@ func (dc *controller) reconcileClusterMachineDeployment(key string) error {
 	// If MachineDeployment is frozen and no deletion timestamp, don't process it
 	if deployment.Labels["freeze"] == "True" && deployment.DeletionTimestamp == nil {
 		glog.V(3).Infof("MachineDeployment %q is frozen, and hence not processeing", deployment.Name)
-		return nil
+		//return nil
 	}
```

Comment:
Do we need this `if` block now? If it's left out for logging, then the log statement should also change to reflect the new logic.

Reply:
Fixed in #115